lib/logging/logging.dart

```diff
@@ -578,12 +608,57 @@ class LogDetailsUI extends CoreElement {
   CoreElement content;
   CoreElement message;
 
+  InspectorTreeHtml tree;
+
   void setData(LogData data) {
     // Reset the vertical scroll value if any.
     content.element.scrollTop = 0;
 
     this.data = data;
 
+    tree = null;
+    if (data.node != null) {
+      message.clear();
+      tree = InspectorTreeHtml(
+        summaryTree: false,
+        treeType: FlutterTreeType.widget,
+        onSelectionChange: () {
+          final InspectorTreeNode node = tree.selection;
+          if (node != null) {
+            tree.maybePopulateChildren(node);
+          }
+          // The ui wasn't updated because this selection was triggered by the logging
```

Comment:
Remove this comment. it is somewhat wrong.

Reply:
👍